sql/2022/markup/doctype.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
+  _TABLE_SUFFIX AS client,
+  LOWER(REGEXP_REPLACE(TRIM(doctype), r' +', ' ')) AS doctype, # remove extra spaces and make lower case
```

Comment:
Should you also remove leading and trailing spaces?

Reply:
There aren't any AFAICT from the [results](https://docs.google.com/spreadsheets/d/1grkd2_1xSV3jvNK6ucRQ0OL1HmGTsScHuwA8GZuRLHU/edit#gid=1535288056)